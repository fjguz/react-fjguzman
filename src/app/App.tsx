import React from 'react';
import PathProvider, { PathContext } from './context/pathContext';
import Page from './pages/Page';

type NavigationProps = {};

const App: React.FC<NavigationProps> = () => {

  return (
    <PathProvider>
      <Page/>
    </PathProvider>
  )
}

export default App;
