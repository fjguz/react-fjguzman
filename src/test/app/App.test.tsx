import renderer from 'react-test-renderer';
import App from '../../app/App';


it('renders correctly', () => {
  const render = renderer.create(<App></App>).toJSON();
  expect(render).toMatchSnapshot();
}); 