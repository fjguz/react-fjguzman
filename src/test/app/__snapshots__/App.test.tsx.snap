// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="welcome"
>
  <main
    className="wrap wrap-main  "
  >
    <header
      className="header-main"
    >
      <div>
        <h1>
          Fran
          <span>
            Guzm√°n
          </span>
        </h1>
      </div>
    </header>
    <section
      className="main-section "
      id="welFront"
      onClick={[Function]}
    >
      <div
        className="wrap-section"
      >
        <header>
          <h2>
            front-end
            <span>
              developer
            </span>
          </h2>
        </header>
        
      </div>
    </section>
    <section
      className="main-section "
      id="welBack"
      onClick={[Function]}
    >
      <div
        className="wrap-section"
      >
        <header>
          <h2>
            back-end
            <span>
              developer
            </span>
          </h2>
        </header>
        
      </div>
    </section>
  </main>
</div>
`;
